{
  "uid" : "1531f52f72d4601",
  "name" : "Verify that links in the Footer lead to correct pages after clicking",
  "fullName" : "tests.footer_test.TestFooterLinks#test_fp_03_02_verify_links_lead_to_the_correct_pages",
  "historyId" : "cf018cf2afd91eeaeb2abb69b7259d6a",
  "time" : {
    "start" : 1728697261737,
    "stop" : 1728697261737,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5592dffd902a <unknown>\n#1 0x5592dfcbf5e0 <unknown>\n#2 0x5592dfcb77eb <unknown>\n#3 0x5592dfca7a09 <unknown>\n#4 0x5592dfca9744 <unknown>\n#5 0x5592dfca7cbd <unknown>\n#6 0x5592dfca752f <unknown>\n#7 0x5592dfca7422 <unknown>\n#8 0x5592dfca5493 <unknown>\n#9 0x5592dfca5ada <unknown>\n#10 0x5592dfcc1f41 <unknown>\n#11 0x5592dfd538c5 <unknown>\n#12 0x5592dfd33b22 <unknown>\n#13 0x5592dfd52d7d <unknown>\n#14 0x5592dfd338c3 <unknown>\n#15 0x5592dfd016b3 <unknown>\n#16 0x5592dfd0268e <unknown>\n#17 0x5592dffa3a2b <unknown>\n#18 0x5592dffa79b1 <unknown>\n#19 0x5592dff90225 <unknown>\n#20 0x5592dffa8532 <unknown>\n#21 0x5592dff7538f <unknown>\n#22 0x5592dffc7f28 <unknown>\n#23 0x5592dffc80f3 <unknown>\n#24 0x5592dffd7e7c <unknown>\n#25 0x7f7f1e494ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b5d3a40bfcf38fc28e51b70649a35914\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4df4f66f00>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5592dffc80f3 \\\\u003Cunknown>\\\\n#24 0x5592dffd7e7c \\\\u003Cunknown>\\\\n#25 0x7f7f1e494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5592dffd902a <unknown>\nE       #1 0x5592dfcbf5e0 <unknown>\nE       #2 0x5592dfcb77eb <unknown>\nE       #3 0x5592dfca7a09 <unknown>\nE       #4 0x5592dfca9744 <unknown>\nE       #5 0x5592dfca7cbd <unknown>\nE       #6 0x5592dfca752f <unknown>\nE       #7 0x5592dfca7422 <unknown>\nE       #8 0x5592dfca5493 <unknown>\nE       #9 0x5592dfca5ada <unknown>\nE       #10 0x5592dfcc1f41 <unknown>\nE       #11 0x5592dfd538c5 <unknown>\nE       #12 0x5592dfd33b22 <unknown>\nE       #13 0x5592dfd52d7d <unknown>\nE       #14 0x5592dfd338c3 <unknown>\nE       #15 0x5592dfd016b3 <unknown>\nE       #16 0x5592dfd0268e <unknown>\nE       #17 0x5592dffa3a2b <unknown>\nE       #18 0x5592dffa79b1 <unknown>\nE       #19 0x5592dff90225 <unknown>\nE       #20 0x5592dffa8532 <unknown>\nE       #21 0x5592dff7538f <unknown>\nE       #22 0x5592dffc7f28 <unknown>\nE       #23 0x5592dffc80f3 <unknown>\nE       #24 0x5592dffd7e7c <unknown>\nE       #25 0x7f7f1e494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728697261737,
      "stop" : 1728697262513,
      "duration" : 776
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728697262513,
      "stop" : 1728697262936,
      "duration" : 423
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5592dffd902a <unknown>\n#1 0x5592dfcbf5e0 <unknown>\n#2 0x5592dfcb77eb <unknown>\n#3 0x5592dfca7a09 <unknown>\n#4 0x5592dfca9744 <unknown>\n#5 0x5592dfca7cbd <unknown>\n#6 0x5592dfca752f <unknown>\n#7 0x5592dfca7422 <unknown>\n#8 0x5592dfca5493 <unknown>\n#9 0x5592dfca5ada <unknown>\n#10 0x5592dfcc1f41 <unknown>\n#11 0x5592dfd538c5 <unknown>\n#12 0x5592dfd33b22 <unknown>\n#13 0x5592dfd52d7d <unknown>\n#14 0x5592dfd338c3 <unknown>\n#15 0x5592dfd016b3 <unknown>\n#16 0x5592dfd0268e <unknown>\n#17 0x5592dffa3a2b <unknown>\n#18 0x5592dffa79b1 <unknown>\n#19 0x5592dff90225 <unknown>\n#20 0x5592dffa8532 <unknown>\n#21 0x5592dff7538f <unknown>\n#22 0x5592dffc7f28 <unknown>\n#23 0x5592dffc80f3 <unknown>\n#24 0x5592dffd7e7c <unknown>\n#25 0x7f7f1e494ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728697262513,
        "stop" : 1728697262936,
        "duration" : 423
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5592dffd902a <unknown>\n#1 0x5592dfcbf5e0 <unknown>\n#2 0x5592dfcb77eb <unknown>\n#3 0x5592dfca7a09 <unknown>\n#4 0x5592dfca9744 <unknown>\n#5 0x5592dfca7cbd <unknown>\n#6 0x5592dfca752f <unknown>\n#7 0x5592dfca7422 <unknown>\n#8 0x5592dfca5493 <unknown>\n#9 0x5592dfca5ada <unknown>\n#10 0x5592dfcc1f41 <unknown>\n#11 0x5592dfd538c5 <unknown>\n#12 0x5592dfd33b22 <unknown>\n#13 0x5592dfd52d7d <unknown>\n#14 0x5592dfd338c3 <unknown>\n#15 0x5592dfd016b3 <unknown>\n#16 0x5592dfd0268e <unknown>\n#17 0x5592dffa3a2b <unknown>\n#18 0x5592dffa79b1 <unknown>\n#19 0x5592dff90225 <unknown>\n#20 0x5592dffa8532 <unknown>\n#21 0x5592dff7538f <unknown>\n#22 0x5592dffc7f28 <unknown>\n#23 0x5592dffc80f3 <unknown>\n#24 0x5592dffd7e7c <unknown>\n#25 0x7f7f1e494ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"b5d3a40bfcf38fc28e51b70649a35914\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728697167335,
      "stop" : 1728697167335,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728697262967,
      "stop" : 1728697263029,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Footer"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "footer_test"
  }, {
    "name" : "subSuite",
    "value" : "TestFooter"
  }, {
    "name" : "host",
    "value" : "fv-az570-490"
  }, {
    "name" : "thread",
    "value" : "2304-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.footer_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 26,
        "unknown" : 0,
        "total" : 30
      },
      "items" : [ {
        "uid" : "61cd4bf418667e6e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/61cd4bf418667e6e",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5571237af02a <unknown>\n#1 0x5571234955e0 <unknown>\n#2 0x55712348d7eb <unknown>\n#3 0x55712347da09 <unknown>\n#4 0x55712347f744 <unknown>\n#5 0x55712347dcbd <unknown>\n#6 0x55712347d52f <unknown>\n#7 0x55712347d422 <unknown>\n#8 0x55712347b493 <unknown>\n#9 0x55712347bada <unknown>\n#10 0x557123497f41 <unknown>\n#11 0x5571235298c5 <unknown>\n#12 0x557123509b22 <unknown>\n#13 0x557123528d7d <unknown>\n#14 0x5571235098c3 <unknown>\n#15 0x5571234d76b3 <unknown>\n#16 0x5571234d868e <unknown>\n#17 0x557123779a2b <unknown>\n#18 0x55712377d9b1 <unknown>\n#19 0x557123766225 <unknown>\n#20 0x55712377e532 <unknown>\n#21 0x55712374b38f <unknown>\n#22 0x55712379df28 <unknown>\n#23 0x55712379e0f3 <unknown>\n#24 0x5571237ade7c <unknown>\n#25 0x7fa9d6c94ac3 <unknown>",
        "time" : {
          "start" : 1728610953085,
          "stop" : 1728610953085,
          "duration" : 0
        }
      }, {
        "uid" : "b9dcb142eeff77af",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/b9dcb142eeff77af",
        "status" : "passed",
        "time" : {
          "start" : 1728524704799,
          "stop" : 1728524718347,
          "duration" : 13548
        }
      }, {
        "uid" : "b4997953105d6857",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/b4997953105d6857",
        "status" : "passed",
        "time" : {
          "start" : 1728438275575,
          "stop" : 1728438394614,
          "duration" : 119039
        }
      }, {
        "uid" : "f46b66859bae0848",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/f46b66859bae0848",
        "status" : "passed",
        "time" : {
          "start" : 1728351949325,
          "stop" : 1728351959063,
          "duration" : 9738
        }
      }, {
        "uid" : "965178a3dc21c18",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/965178a3dc21c18",
        "status" : "passed",
        "time" : {
          "start" : 1728266011812,
          "stop" : 1728266022957,
          "duration" : 11145
        }
      }, {
        "uid" : "77e01725f5670012",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/77e01725f5670012",
        "status" : "passed",
        "time" : {
          "start" : 1728179500490,
          "stop" : 1728179509415,
          "duration" : 8925
        }
      }, {
        "uid" : "469f253d99129194",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/469f253d99129194",
        "status" : "passed",
        "time" : {
          "start" : 1728092691341,
          "stop" : 1728092702529,
          "duration" : 11188
        }
      }, {
        "uid" : "9b3d85a2c7acc413",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/9b3d85a2c7acc413",
        "status" : "passed",
        "time" : {
          "start" : 1728006394889,
          "stop" : 1728006404745,
          "duration" : 9856
        }
      }, {
        "uid" : "1936cd27c0675caf",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/1936cd27c0675caf",
        "status" : "passed",
        "time" : {
          "start" : 1727919988539,
          "stop" : 1727919999809,
          "duration" : 11270
        }
      }, {
        "uid" : "bb37a69735664c77",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/bb37a69735664c77",
        "status" : "passed",
        "time" : {
          "start" : 1727833502194,
          "stop" : 1727833509907,
          "duration" : 7713
        }
      }, {
        "uid" : "e7f8dcc328631e1f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/e7f8dcc328631e1f",
        "status" : "failed",
        "statusDetails" : "AssertionError: Links in the Footer lead to incorrect pages after clicking or did not loaded during the allotted time\nassert False\n +  where False = all(<generator object TestFooter.TestFooterLinks.test_fp_03_02_verify_links_lead_to_the_correct_pages.<locals>.<genexpr> at 0x7fecf1cc7e00>)",
        "time" : {
          "start" : 1727747638717,
          "stop" : 1727747647279,
          "duration" : 8562
        }
      }, {
        "uid" : "27c2c16dcdc05434",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/27c2c16dcdc05434",
        "status" : "failed",
        "statusDetails" : "AssertionError: Links in the Footer lead to incorrect pages after clicking or did not loaded during the allotted time\nassert False\n +  where False = all(<generator object TestFooter.TestFooterLinks.test_fp_03_02_verify_links_lead_to_the_correct_pages.<locals>.<genexpr> at 0x7f4e719ecfb0>)",
        "time" : {
          "start" : 1727660921125,
          "stop" : 1727660930902,
          "duration" : 9777
        }
      }, {
        "uid" : "1617c421fce1b5c1",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/1617c421fce1b5c1",
        "status" : "passed",
        "time" : {
          "start" : 1727574732612,
          "stop" : 1727574741916,
          "duration" : 9304
        }
      }, {
        "uid" : "7c477bd96eb68a05",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/7c477bd96eb68a05",
        "status" : "passed",
        "time" : {
          "start" : 1727487973277,
          "stop" : 1727487981992,
          "duration" : 8715
        }
      }, {
        "uid" : "dc301b3ec5eaac29",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/dc301b3ec5eaac29",
        "status" : "passed",
        "time" : {
          "start" : 1727401606242,
          "stop" : 1727401648255,
          "duration" : 42013
        }
      }, {
        "uid" : "e529dc689a259f14",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/e529dc689a259f14",
        "status" : "passed",
        "time" : {
          "start" : 1727315130595,
          "stop" : 1727315141285,
          "duration" : 10690
        }
      }, {
        "uid" : "2466d187329588df",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/2466d187329588df",
        "status" : "passed",
        "time" : {
          "start" : 1727228801749,
          "stop" : 1727228812750,
          "duration" : 11001
        }
      }, {
        "uid" : "2b83e3e764e0ccd5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/2b83e3e764e0ccd5",
        "status" : "passed",
        "time" : {
          "start" : 1727142355255,
          "stop" : 1727142365591,
          "duration" : 10336
        }
      }, {
        "uid" : "589a305773f7c624",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/589a305773f7c624",
        "status" : "passed",
        "time" : {
          "start" : 1727055897468,
          "stop" : 1727055906213,
          "duration" : 8745
        }
      }, {
        "uid" : "78f6813cefa85255",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/40//#testresult/78f6813cefa85255",
        "status" : "passed",
        "time" : {
          "start" : 1726969825812,
          "stop" : 1726969833513,
          "duration" : 7701
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1531f52f72d4601.json",
  "parameterValues" : [ ]
}